scss/mixins/_grid.scss

```
@@ -18,6 +18,16 @@
   margin-left: -$gutter / 2;
 }
 
+// For each breakpoint, define the maximum width of the container in a media query
+@mixin make-container-max-widths($max-widths: $container-max-widths, $breakpoints: $grid-breakpoints) {
+  @each $breakpoint, $container-max-width in $max-widths {
+    @include media-breakpoint-up($breakpoint, $breakpoints) {
+      max-width: $container-max-width;
+    }
+  }
+  @include deprecate("The `make-container-max-widths` mixin", "v4.5.2", "v5", $ignore-warning);
```

`$ignore-warning` is undefined, thus can be removed:

```suggestion
  @include deprecate("The `make-container-max-widths` mixin", "v4.5.2", "v5");
```

---

True, bad CP.